helmfile.d/helmfile-05.init.yaml

```diff
@@ -32,11 +32,3 @@ releases:
     labels:
       pkg: keycloak
     <<: *raw
-  - name: keycloak
-    installed: true
-    namespace: keycloak
-    labels:
-      pkg: keycloak
-      app: core
```

Comment:
You do not need to change the order. Just remove the  app: core label. By doing it you delegate installation solely to argocd.

Reply:
Good. I moved this back now.